src/Microsoft.Security.Utilities.Core/RegexPattern.cs

```diff
@@ -312,13 +315,14 @@ public static string GenerateSha256Hash(string text)
 
     public virtual Tuple<string, string>? GetMatchIdAndName(string match) => new Tuple<string, string>(Id, Name);
 
-    [DataMember(Order = 4)]
+    [DataMember(Order = 5)]
     public string Pattern { get; protected set; }
 
     /// <summary>
     /// Gets or sets an opaque, stable identifier for the pattern (corresponding to a SARIF 'reportingDescriptorReference.id' value).
     /// </summary>
     [DataMember(Order = 1)]
+
```

Comment:
This blank line is really good at coming back, lol.

Reply:
nuts fixed